src/audio-player-composable/components/volume-control/types.ts

```diff
@@ -7,15 +7,15 @@ export interface AudioPlayerVolumeControlOverridesProps {
   button?: {
     stylePreset?: MQ<string>;
     iconSize?: string;
-    size?: ButtonSize;
   };
 }
 
 export interface AudioPlayerVolumeControlProps {
   vertical?: boolean;
   collapsed?: boolean;
   overrides?: AudioPlayerVolumeControlOverridesProps;
-  muteKeyboardShortcuts?: string[] | string;
+  keyboardShortcuts?: {muteButton: string | string[]};
```

Comment:
what about calling this just mute
```suggestion
  keyboardShortcuts?: {mute: string | string[]};
```

Reply:
`muteToggle` ?

Reply:
Done